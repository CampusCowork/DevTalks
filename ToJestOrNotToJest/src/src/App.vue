<template>
  <div id="app">
    <addr></addr>
  </div>
</template>

<script>
import Addr from './components/Addr'

export default {
  name: 'app',

  components: {
    Addr
  }
}
</script>

<style>
@import 'assets/styles/tailwind.postcss';

#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>
