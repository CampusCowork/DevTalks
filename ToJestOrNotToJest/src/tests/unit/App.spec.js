import { mount } from '@vue/test-utils'
import App from '@/App.vue'

describe('App.vue', () => {
  it('renders the app component', () => {
    const wrapper = mount(App)
    expect(wrapper.element).toMatchSnapshot()
  })
})
