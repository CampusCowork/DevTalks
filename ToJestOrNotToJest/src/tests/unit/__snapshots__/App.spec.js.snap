// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders the app component 1`] = `
<div
  id="app"
>
  <div>
    <h1
      class="logo text-purple text-5xl"
    >
      
    ADDR+
  
    </h1>
     
    <div
      class="addr flex mx-auto items-center justify-center mt-8"
    >
      <div
        class="addr-input"
      >
        <input
          class="px-4 py-2 w-32 bg-grey-lighter outline-none"
          data-cy="input"
          type="text"
        />
      </div>
       
      <span
        class="font-bold mx-4"
      >
        +
      </span>
       
      <div
        class="addr-input"
      >
        <input
          class="px-4 py-2 w-32 bg-grey-lighter outline-none"
          data-cy="input"
          type="text"
        />
      </div>
       
      <span
        class="font-bold mx-4"
      >
        =
      </span>
       
      <div
        class="addr-result"
      >
        <span
          data-cy="result"
        >
          
    0
  
        </span>
      </div>
    </div>
  </div>
</div>
`;
